web/gulpfile.babel.js

```diff
@@ -47,25 +30,13 @@ gulp.task('eslint-ci', () => {
   // Exit process with an error code (1) on lint error for CI build.
   return runEslint().pipe(eslint.failAfterError());
 });
-//
-// gulp.task('karma-ci', (done) => {
-//   runKarma({singleRun: true}, done);
-// });
-//
-// gulp.task('karma', (done) => {
-//   runKarma({singleRun: false}, done);
-// });
 
 gulp.task('test', (done) => {
-  // TODO: Fix 'karma-ci'
-  runSequence('eslint-ci', 'build-webpack', done);
+  runSequence('eslint-ci', 'build', done);
 });
 
-gulp.task('server', ['env', 'build'], bg('node', './src/server'));
+gulp.task('server-hot', bg('node', './webpack/server'));
 
-// gulp.task('tdd', (done) => {
-//   // Run karma configured for TDD.
-//   runSequence('server', 'karma', done);
-// });
+gulp.task('server', ['env', 'server-hot'], bg('nodemon', './src/server'));
 
```

Comment:
It will work this way :) When command is executed with `npm run` `./node_modules/.bin` is automatically added to path